main.js

```diff
@@ -0,0 +1,142 @@
+    img: 'img/img2.png',
+    companyName: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    year: 2015,
+    decription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
+
+  project3: {
+    name: 'Facebook 360',
+    img: 'img/img3.png',
+    companyName: 'FACEBOOK',
+    job: 'Full Stack Dev',
+    year: 2015,
+    decription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
+  project4: {
+    name: 'Uber Navigation',
+    img: 'img/img4.png',
+    companyName: 'Uber',
+    job: 'Lead Developer',
+    year: 2018,
+    decription:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor  ',
+    tags: ['html', 'Ruby on rails', 'css', 'javaScript'],
+  },
+};
+
+const nameID = document.getElementById('name');
+const img = document.getElementById('img');
+const companyName = document.getElementById('companyName');
+const job = document.getElementById('job');
+const year = document.getElementById('data');
+const descrition = document.getElementById('description');
+const tags = document.getElementById('tags');
+const button1 = document.querySelector('#btn-1');
+const button2 = document.querySelector('#btn-2');
+const button3 = document.querySelector('#btn-3');
+const button4 = document.querySelector('#btn-4');
+const btnClosePop = document.querySelector('.popup-close-btn');
+const popupOverlay = document.querySelector('.popup-overlay');
+const popup = document.getElementById('popup');
+
+function openPopUp() {
+  popup.classList.remove('hidden');
+}
+
+function changePopup(order) {
+  tags.innerHTML = '';
+
+  const project = data[`project${order}`];
+  nameID.innerHTML = project.name;
+  img.src = project.img;
+  companyName.innerHTML = project.companyName;
+  job.innerHTML = project.job;
+  year.innerHTML = project.year;
+  descrition.innerHTML = project.decription;
+  const tagsLength = project.tags.length;
+  for (let i = 0; i < tagsLength; i += 1) {
+    const element = document.createElement('li');
+    element.innerHTML = `${project.tags[i]}`;
+    tags.appendChild(element);
+  }
+  openPopUp();
+}
+
+function closePopUp() {
+  popup.classList.add('hidden');
+}
+
+button1.addEventListener('click', () => {
+  openPopUp();
+  changePopup(1);
+});
+
+button2.addEventListener('click', () => {
+  openPopUp();
+  changePopup(2);
+});
+
+button3.addEventListener('click', () => {
+  openPopUp();
+  changePopup(3);
+});
+
+button4.addEventListener('click', () => {
+  openPopUp();
+  changePopup(4);
+});
+
+popupOverlay.addEventListener('click', closePopUp);
+btnClosePop.addEventListener('click', closePopUp);
+
+const form = document.getElementById('contact-form');
+const email = document.getElementById('email');
+const msg = document.querySelector('.show-msg');
+
+function validationEmail(e) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.